lib/SQL/Translator/Producer/Oracle.pm

```diff
@@ -668,19 +655,20 @@ sub create_field {
         push @trigger_defs, $trigger;
     }
 
-    if ( lc $field->data_type eq 'timestamp' ) {
-        my $base_name = $table_name . "_". $field_name;
-        my $trig_name = quote(mk_name( $base_name, 'ts' ), $qt);
-        my $trigger =
-          "CREATE OR REPLACE TRIGGER $trig_name\n".
-          "BEFORE INSERT OR UPDATE ON $table_name_q\n".
-          "FOR EACH ROW WHEN (new.$field_name_q IS NULL)\n".
-          "BEGIN\n".
-          " SELECT sysdate INTO :new.$field_name_q FROM dual;\n".
-          "END;\n";
-
-          push @trigger_defs, $trigger;
-    }
+    # Do not create a trigger to insert sysdate to all timestamp fields
+    # if ( lc $field->data_type eq 'timestamp' ) {
```

Comment:
I don't think it's good practice to leave slabs of commented code. If you want to delete it, delete it.

Reply:
I agree, I just wasn't sure how things were being handled on this project. I wanted that change and its reason to be obvious. I will get rid of that.

Reply:
Fixed in commit [b14da74](https://github.com/dbsrgits/sql-translator/pull/143/commits/b14da74a5bb7e2e342e2651c6abfdf656e153128)